pkg/channel/event_dispatcher_test.go

```diff
@@ -0,0 +1,498 @@
+/*
+Copyright 2018 The Knative Authors
```

Comment:
<!--copyright-bot-->
Incorrect copyright year:
```suggestion
Copyright 2019 The Knative Authors
```

Reply:
old file message_dispatcher_test that was renamed... do you know if we should change the copyright @Harwayne ?